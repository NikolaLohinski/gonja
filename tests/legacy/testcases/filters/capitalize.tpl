{{ ""|capitalize }}
{{ 5|capitalize }}
{{ "h"|capitalize }}
{{ "hello there!"|capitalize }}
{{ "HELLO THERE!"|capitalize }}
{{ simple.chinese_hello_world|capitalize }}
