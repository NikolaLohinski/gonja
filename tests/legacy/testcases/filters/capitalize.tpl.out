

H
Hello there!
Hello there!
你好世界
